tools/markdown-checker/index.js

```diff
@@ -0,0 +1,14 @@
+const readMarkdown = require("./lib/parser/readMarkdown");
+const parseToAST = require("./lib/parser/parseToAst");
+const collectLinkDefinitions = require("./lib/analyzer/collectLinkDefinitions");
+const tableAnalyzer = require("./lib/analyzer/analyzeTable");
+const config = require("./config.json");
+
+function processStage3({ stage3 } = config) {
+  const markdownStage3 = readMarkdown(stage3);
+  const parsedFile = parseToAST(markdownStage3);
+  const collectedLinkDefinitions = collectLinkDefinitions(parsedFile);
+  const tableStage1 = tableAnalyzer(parsedFile, collectedLinkDefinitions);
```

Comment:
`tableStage1` is not used anywhere. Also why stage 1?

Reply:
Still developing...